bench/orphaned_process_test.exs

```diff
@@ -0,0 +1,153 @@
+# This isn't really a benchmark, but I wasn't sure where to put it.
+# This script sets up several different pools and then runs a battery
+# of tests for kicking off, killing, and crashing a bunch of different jobs.
+# Throughout that process is periodically checks to make sure that all
+# of the pools have cleaned up their states correctly
```

Comment:
I'm wondering why this is not just an automated test instead of just a plain elixir script. Is it just too slow because of the :timer.sleep calls?